jaiqu/helpers.py

```diff
@@ -57,17 +57,17 @@ def identify_key(key, value, input_schema, key_hints=None) -> tuple[Optional[str
         "content": f"Is `{key}` of type `{value}` present in the desired schema?:\n\n  {input_schema}"
     }]
 
-    reasoning_response = OpenAI().chat.completions.create(messages=messages,
+    reasoning_response = OpenAI(api_key).chat.completions.create(messages=messages,
                                                           model="gpt-4",
                                                           #                                                         logit_bias={2575: 100, 4139: 100},
                                                           #                                                         max_tokens=1
                                                           )
     completion = str(reasoning_response.choices[0].message.content)
 
-    return (to_key(completion), completion)
+    return to_key(completion), completion
```

Comment:
I prefer this change but your formatter removed the paren and we should turn off our formatters/linters and instead have it done by commit hook. To avoid it looking like you changed lines that you didn't